app/controllers/case_logs_controller.rb

```diff
@@ -26,6 +26,7 @@ def submit_form
     previous_page = params[:case_log][:previous_page]
     questions_for_page = form.questions_for_page(previous_page).keys
     answers_for_page = page_params(questions_for_page).select { |k, _v| questions_for_page.include?(k) }
+    @case_log.custom_validator_options = { previous_page: previous_page, validation: form.questions_for_page(previous_page)[previous_page]["regex_validation"] }
```

Comment:
This feels like a slight OOP anti-pattern, could we make this a method on CaseLog?